### PR TITLE
closes #154

### LINKED ISSUE #154: #1123
Breadcrumb tests

### PR DESCRIPTION
**Pull request by kdorer**

### CODE REVIEW

**Comment by bammons** [comment]
For some reason this push deleted all of lincoln's work on sources

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV['RAILS_ENV'] ||= 'test'
-require 'coveralls'
-Coveralls.wear!('rails')
+# require 'coveralls'
+# Coveralls.wear!('rails')
```
Comment (bammons) [inline spec/spec_helper.rb]: need to uncomment this

**Comment by bammons** [comment]
+1

**Comment by Linkster700** [comment]
+1

**Pull request merged**